### DISCUSSION TITLE
Add C# and .NET as a scripting language.

### DISCUSSION THREAD

**Discussion by icecactus** [windmill-labs/windmill#4449] [Questions] [answered]
Love windmill and I love c# and .NET. Would be awesome to be able to use them together. Anything on the horizon for making this happen?

**Comment by rubenfiszel** [comment, marked as answer]
Yes C#, Ruby, R and Java are planned for the next 2 months

**Reply by ajaya** [reply]
HI.  Any update on Ruby support?

**Comment by icecactus** [comment]
Great! Was not expecting that response...figured it would never happen. Made my day lol!
